test-data/unit/check-isinstance.test

```diff
@@ -602,6 +638,14 @@ while 1:
 else:
     x + 1
 x + 1      # E: Unsupported operand types for + (likely involving Union)
+x = 1
+for y in [1]:
+    x + 1
+    x = 'a'
+    break
+else:
+    x + 1
```

Comment:
Yeah, *Loop4, I believe.  (It's on 'if 1:', but a bunch of my tests assume mypy isn't clever enough to realize such branches are always taken.)